code/game/objects/structures/crates_lockers/__closets.dm

```diff
@@ -434,10 +434,6 @@
 			breakout = 0
 			return
 
-			playsound(src, breakout_sound, 100, 1)
-			animate_shake()
-			add_fingerprint(escapee)
```

Comment:
Huh, I always wondered what broke the animation to break out of lockers. Could this code be fixed by just having the proper number of tabs?

Reply:
Locker animation has been fixed now.